db/db.go

```diff
@@ -102,18 +102,14 @@ func get(ctx context.Context, db *sql.DB, query string, args ...interface{}) (st
 
 	buf := []byte{}
 	err := row.Scan(&buf)
-	if err == nil {
-		return string(buf), nil
-	}
-
-	if err != sql.ErrNoRows {
-		err = errors.Wrap(err, "SELECT")
+	if err != nil {
+		if err == sql.ErrNoRows {
+			return "", err // if sql.ErrNoRows, return error as is
```

Comment:
we should wrap this error

Reply:
oops i missed this comment. The reason I didn't wrap this error was because `sql.ErrNoRows` is checked here: https://github.com/tinkerbell/tink/blob/71a1c33b0753bec7f501e00cd64e882742f286e6/db/workflow.go#L112

If I wrap the error, it becomes harder to check (unless you have any suggestions).

Reply:
ah true